math/fibonacci_matrix_exponentiation.cpp

```diff
@@ -0,0 +1,143 @@
+ /**
+ * @file fibonacci_matrix_exponentiation.cpp
+ * @brief Computes N^th Fibonacci number given as
+ * input argument
+ *
+ * Takes O(logn) time to compute nth Fibonacci number
+ * 
+ *
+ * \author [villayatali123](https://github.com/villayatali123)
+ * \author [unknown author]()
+ * @see fibonacci.cpp, fibonacci_fast.cpp, string_fibonacci.cpp, fibonacci_large.cpp
+ * @details
+ *Begin 
```

Comment:
If you can review the checklist, one of the items is about the documentation. It explains what protocol we follow: doxygen. In this protocol, all documentations will be interpreted in Markdown syntax. These successive lines, though look good in the code, will not render properly i.e., they will not be rendered as WYSIWYG. To add breaks, add a new line "\n" at the end of each line. Please review the existing codes, references, checklist.